### DISCUSSION TITLE
Add Enable Minimize To Icon on new Elementary OS 8

### DISCUSSION THREAD

**Discussion by patx** [pantheon-tweaks/pantheon-tweaks#250] [Ideas] [answered]
With OS 8 comes Wayland and a new dock. This is very cool and a long time coming. The developers at eOS are pushing a workspace centrist desktop and have made the dock to not minimize and open application when clicking on its Icon in the current workspace. We need a tweak that can enable this once more!

**Comment by ryonakano** [comment, marked as answer]
You might imagine adding a kind of a switch to enable/disable the feature of minimizing to Dock in Pantheon Tweaks, but it's impossible unless Dock implements the feature itself. Pantheon Tweaks is just a tool to tweak preferences that Pantheon doesn't provide a UI to do that.

**Reply by patx** [reply]
ah well worth a shot... sucks they took this feature out of the dock

**Reply by doobes** [reply]
There have been a number of unpleasant regressions in v8.0.
